server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/copybooks/FlavorType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.preprocessor.delegates.copybooks;
+
+/**
+ * Represents supported copybook flavor types, used for resolving copybooks for different flavors
+ */
+public enum FlavorType {
+  Idms,
+  Maid
```

Comment:
According to the Java naming convention, the enum elements should be written in the upper case.